### PR TITLE
MRG: create files for dynmics pattern project and address issue #139

### PR DESCRIPTION
**Pull request by HongfeiZhao**
Create pull request for adding current code: create time matrix.
@harryh1993  @Chenyu-Renee @lynnzhao92 @mingyungkim @wyshi

### CODE REVIEW

**Comment by harryh1993** [comment]
@HongfeiZhao I have just finished reviewing this part. Ready to merge.

**Comment by jarrodmillman** [comment]
This doesn't have any tests.  You also need to have all pull requests reviewed and improved (i.e., a review is more involved than someone saying it is ready to go.)  I've changed this back to WIP.

**Comment by harryh1993** [comment]
@jarrodmillman I have discussed the issues with jack (Hongfei) in person, such as the variable names and output styles etc. We just think it is more efficient to directly contact each other through phone/text instead of leaving messages here. I will also mention the issues here next time I review a pull request

**Comment by harryh1993** [comment]
Just a thought, you might add a parameter to restrict the number of columns of the output, so that when we run the test file it will not take 10 minutes to run the test

**Comment by HongfeiZhao** [comment]
please review my code. @lynnzhao92 @mingyungkim

File: mousestyles/dynamics/__init__.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    Returns
+    -------
+    time: Pandas.DataFrame
+        a matrix represents the activity for a certain
+        mouse day and a certain time.
+
+    Examples
+    --------
+    >>> time = create_time_matrix(combined_gap=4, time_gap=1).iloc[0, 0:10]
+    >>> strain    0
```
Comment (lynnzhao92) [inline mousestyles/dynamics/__init__.py]: Inside of the example, strain 0 mouse 0 day 0, does that mean it is for the first mouse of the first strain and for the first mouse day?

File: mousestyles/dynamics/__init__.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+from mousestyles import data
+
+
+def create_time_matrix(combined_gap=4, time_gap=1, days_index=137):
+    r"""
+    Return a time matrix for estimate the MLE parobability.
+    The rows are 137 mousedays. The columns are time series
+    in a day. The data are the mouse activity at that time.
+    0 represents IS, 1 represents eating, 2 represents
+    drinking, 3 represents others activity in AS.
```
Comment (mingyungkim) [inline mousestyles/dynamics/__init__.py]: Grammar issue: "others activity" should be "other activities"

File: mousestyles/dynamics/__init__.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        a matrix represents the activity for a certain
+        mouse day and a certain time.
+
+    Examples
+    --------
+    >>> time = create_time_matrix(combined_gap=4, time_gap=1).iloc[0, 0:10]
+    >>> strain    0
+        mouse     0
+        day       0
+        48007     0
```
Comment (lynnzhao92) [inline mousestyles/dynamics/__init__.py]: Do we need to figure out what the time index means?

**Pull request merged**